src/js/main.js

```diff
@@ -6,7 +6,10 @@ const availableHotelItems = document.getElementById('available-hotels_items')
 buttonSearch.addEventListener('click', async function (event) {
   event.preventDefault();
   const inputSity = document.getElementById('city').value.toUpperCase();
-  let url = `https://if-student-api.onrender.com/api/hotels?search=${inputSity}`;
+  let url = new URL('https://if-student-api.onrender.com/api/hotels');
+  //const param = new URLSearchParams();
```

Comment:
Please remove all code comments. I mean comments like this:
```
//const param = new URLSearchParams();
//console.log(data)
```
etc...

But you can leave comments like 
```
// домашка 12,16
//ограничения по количеству + переключение кнопок +-
//селекты
```
But, in my opinion, comments should be written in English, like code.